.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Install kubectl
+      run: |
+        set -euo pipefail
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl        
+
+    - name: Install krew
+      shell: bash
+      run: |
+        set -euo pipefail
+        cd "$(mktemp -d)" &&
+        curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}" &&
+        tar zxvf krew.tar.gz &&
+        ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
+        
+    - name: Install validate-krew-manifest plugin
+      run : |
+        set -euo pipefail
+        export GOBIN=$HOME/bin
+        go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
+        
+    - name: Validate updated plugin manifests
+      run : |
+        set -euo pipefail
```

Comment:
It worries me that this fails, if there are no plugins in the changeset. Can you try to run with `set -x` to see what are the exact commands?

Reply:
I did, [here](https://github.com/alonyb/krew-index/runs/409226155?check_suite_focus=true) is the result. It didn't work

Reply:
:thinking: The run you linked to seems to be a version with `grep` and has a change in `access-matrix.yaml`. Have you tried the variant without `grep`:
```sh
git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.sha }} -- plugins/ |
          tee /dev/stderr |
          xargs -I _ $HOME/bin/validate-krew-manifest -manifest _
```

Let's run this with `set -x` and see if it fails.

Reply:
@corneliusweig 
It looks perfect, [here](https://github.com/alonyb/krew-index/runs/410034531?check_suite_focus=true#step:7:18) is the log.